WordPress/Sniffs/VIP/FileSystemWritesDisallowSniff.php

```diff
@@ -102,4 +118,35 @@ public function getGroups() {
 		return $groups;
 	}
 
+	/**
+	 * This sniff is active, but it's deprecated, handle the deprecation notices
+	 *
+	 * @deprecated 1.0.0 Added to allow for throwing the deprecation notices.
+	 *
+	 * @param int $stackPtr The position of the current token in the stack.
+	 *
+	 * @return void|int
+	 */
+	public function process_token( $stackPtr ) {
+		if ( false === $this->thrown['DeprecatedSniff'] ) {
+			$this->thrown['DeprecatedSniff'] = $this->phpcsFile->addWarning(
+				'The "WordPress.VIP.FileSystemWritesDisallowSniff" sniff has been deprecated. Please update your custom ruleset.',
```

Comment:
`WordPress.VIP.FileSystemWritesDisallowSniff` => `WordPress.VIP.FileSystemWritesDisallow` (no `Sniff` at the end).

Please adjust this here and everywhere else in the deprecation warnings.